blazor/diagram/user-handle.md

```diff
@@ -424,6 +424,88 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ![VisibleTarget](Images/blazor-diagram-user-handle-visible-target.gif)
 
+### How to provide a template to userhandle
+
+You can define user handle style using template in [UserHandleTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTemplates.html#Syncfusion_Blazor_Diagram_DiagramTemplates_UserHandleTemplate) at tag level. The template will be rendered when the PathData and Source properties of the userhandle are not defined. However, if either PathData or Source is defined, then template will not be rendered as they take precedence. The following code explains how to define template for fixeduserhandle.
```

Comment:
You can define user handle style using a template in the [UserHandleTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTemplates.html#Syncfusion_Blazor_Diagram_DiagramTemplates_UserHandleTemplate) at the tag level. The template will be rendered when the PathData and Source properties of the userhandle are not defined. However, if either PathData or Source is defined, the template will not be rendered as they take precedence. The following code explains how to define a template for the fixeduserhandle.